app/controllers/flood_risk_engine/enrollments/steps_controller.rb

```diff
@@ -19,6 +19,7 @@ def show
         render :show, locals: locals
       end
 
+      # rubocop:disable Metrics/AbcSize
```

Comment:
I think this needs to be re-enabled after the method. Alternatively, move the error handling out into a separate method.